lib/miq_automation_engine/models/miq_ae_domain.rb

```diff
@@ -170,11 +170,16 @@ def squeeze_priorities
   def self.any_enabled?
     MiqAeDomain.enabled.count > 0
   end
+  private_class_method :any_enabled?
```

Comment:
we call this with a receiver in the toolbar_builder.rb, so this method should be moved before the private keyword in a followup PR, so drop this change.

```
git grep any_enabled
app/helpers/application_helper/toolbar_builder.rb:    MiqAeDomain.any_enabled?
lib/miq_automation_engine/models/miq_ae_domain.rb:  def self.any_enabled?
lib/miq_automation_engine/models/miq_ae_domain.rb:  private_class_method :any_enabled?
```